<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewQrCodeDialog</class>
 <widget class="QDialog" name="NewQrCodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pB_generateQR">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>GENERA</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>312</width>
     <height>163</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>NOME:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lE_Name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>COGNOME:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lE_Surname"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>EMAIL:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lE_email"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>INIZIO VALIDITA':</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="startingDateEdit">
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="startingTimeEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>FINE VALIDITA':</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="expiringDateEdit">
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="expiringTimeEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
